config.yaml

```diff
@@ -35,14 +35,19 @@ tasks:
     task_description: "Retrieval is the task of finding relevant documents for a query."
   STS:
     icon: "☘️"
-    metric: spearman
+    metric: cosine_spearman
     metric_description: "Spearman correlation based on the model's similarity metric (usually cosine)"
     task_description: "Semantic Textual Similarity is the task of determining how similar two texts are."
   Summarization:
     icon: "📜"
-    metric: spearman
+    metric: cosine_spearman
```

Comment:
If the current code allows submitting results of models with other distance metrics, then maybe yes; @KennethEnevoldsen probably knows best?

Reply:
For summarization:
```
            "pearson"
            "spearman"
            "cosine_spearman"
            "cosine_pearson"
            "dot_spearman"
            "dot_pearson"
```
I checked main_score for summarization tasks and they have `cosine_spearman`

Reply:
spearman will often just be cosine spearman, but I think it is nicer to leave to up to the model developer to choose their comparison metric. I.e. would leave it as `spearman`

Reply:
But the issue that metrics will filter based on metric specied in cobfig, but in results there is no metric with name `spearman`. I can extend metrics in results file to avoid this, but I don't know if it good solution

Reply:
If this is the case I am fine with keeping it as `cosine_spearman` (we can make a change to custom similarity metrics at a later point)